kusto/ingest/ingest.go

```diff
@@ -64,12 +64,17 @@ type Ingestion struct {
 
 // New is the constructor for Ingestion.
 func New(client *kusto.Client, db, table string) (*Ingestion, error) {
+	return NewWithBlobProperties(client, db, table, filesystem.BlockSize, filesystem.Concurrency)
+}
+
+// NewWithBlobProperties is an advanced constructor for Ingestion, to use in special cases where you need more control over memory consumption.
```

Comment:
Not a fan of the name - it does not really explain what this new command does.
Nor the parameters which need to be explained in the comment.
Do you have the information or can you dig up a better explanation for the user?

Reply:
Let's remove this completely, there is no reason to have a new constructor.